pkg/agent/core/ngt/handler/grpc/object_test.go

```diff
@@ -1337,6 +1337,118 @@ func Test_server_StreamGetObject(t *testing.T) {
 	}
 }
 
+func Test_server_GetTimestamp(t *testing.T) {
+	t.Parallel()
+
+	defaultConfig := config.NGT{
+		Dimension:           100,
+		DistanceType:        "l2",
+		ObjectType:          "float",
+		BulkInsertChunkSize: 10,
+		CreationEdgeSize:    20,
+		SearchEdgeSize:      10,
+		EnableProactiveGC:   false,
+		EnableCopyOnWrite:   false,
+		KVSDB: &config.KVSDB{
+			Concurrency: 10,
+		},
+		BrokenIndexHistoryLimit: 1,
+	}
+
+	setup := func(t *testing.T) (context.Context, Server) {
+		t.Helper()
+		ngt, err := service.New(&defaultConfig)
+		require.NoError(t, err)
+
+		ctx := context.Background()
+		eg, ectx := errgroup.New(ctx)
+		opts := []Option{
+			WithIP(net.LoadLocalIP()),
+			WithNGT(ngt),
+			WithErrGroup(eg),
+		}
+		s, err := New(opts...)
+		require.NoError(t, err)
+
+		return ectx, s
+	}
+
+	type test struct {
+		name     string
+		testfunc func(t *testing.T)
+	}
+
+	tests := []test{
+		{
+			name: "succeeds to get object meta",
+			testfunc: func(t *testing.T) {
```

Comment:
I think It is better to separate the setup logic(`Insert, CreateIndex`..etc) for testing from the comparison by actual testing to improve readability and maintainability. 🤔

Reply:
With the same reason as above, this preliminary setup is also part of the test so I don't think it should be separated. But I added a comment to make it clearer.
[6e3c150](https://github.com/vdaas/vald/pull/2234/commits/6e3c150fae457b1ec9578a47961711d8c500406b)